pages/settings/profile.tsx

```diff
@@ -124,8 +124,11 @@ function SettingsView(props: ComponentProps<typeof Settings> & { localeProp: str
     }).catch((e) => {
       console.error(`Error Removing user: ${props.user.id}, email: ${props.user.email} :`, e);
     });
-    // signout;
-    signOut({ callbackUrl: "/auth/logout" });
+    if (process.env.BASE_URL === "https://app.cal.com/") {
```

Comment:
```suggestion
    if (process.env.NEXT_PUBLIC_BASE_URL === "https://app.cal.com") {
```

Env variables without `NEXT_PUBLIC_` aren't available in the front end.